omod/src/main/webapp/tag.jsp

```diff
@@ -0,0 +1,21 @@
+<%@ include file="/WEB-INF/template/include.jsp"%>
+
+<%@ include file="/WEB-INF/template/header.jsp"%>
+
+<h2><spring:message code="tag-omod.title" /></h2>
```

Comment:
the files on the web side came by default, since i was working on the API, i missed these. I'll get rid of all of them